app-admin/yadm/yadm-3.1.1.ebuild

```diff
@@ -0,0 +1,75 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{8..9} )
+inherit bash-completion-r1 python-any-r1
+
+DESCRIPTION="A dotfile manager for the config files in your home folder"
+HOMEPAGE="https://github.com/TheLocehiliosan/yadm/"
+SRC_URI="https://github.com/TheLocehiliosan/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
```

Comment:
missing `+` here.

Reply:
I don't see yadm saying that license is GPL-3.0 or any later anywhere. Thus, I think it's incorrect for us to assume that. WDYT?

Reply:
it's usually in the file header, not license file itself:
https://github.com/TheLocehiliosan/yadm/blob/3d82aff3e8140361fca18f664fffa64dfd33296e/yadm#L8

Reply:
Oh. Indeed. Thanks!